js/entry2e.js

```diff
@@ -1,4 +1,6 @@
 import { apple2 } from './main2e';
 import * as UI from './ui/apple2';
 
+window.e = true;
```

Comment:
Is this actually still used anywhere?

Reply:
Well, it was, but just in the file dialog and I figured out how to eliminate it.